cvat-ui/src/components/annotation-page/standard-workspace/canvas-wrapper.tsx

```diff
@@ -4,20 +4,26 @@
 
 import React from 'react';
 import { GlobalHotKeys, KeyMap } from 'react-hotkeys';
-import Slider, { SliderValue } from 'antd/lib/slider';
-import Layout from 'antd/lib/layout';
-import Icon from 'antd/lib/icon';
-import Tooltip from 'antd/lib/tooltip';
 
-import { LogType } from 'cvat-logger';
-import { Canvas } from 'cvat-canvas';
-import getCore from 'cvat-core';
+import {
+    Layout,
+    Slider,
+    Icon,
+    Tooltip,
+} from 'antd';
+
```

Comment:
Let's import exact components in future code
Not from ``antd`` wrapper
It common recommendation to reduce bundle size